crates/re_viewer/src/ui/welcome_screen/mod.rs

```diff
@@ -148,7 +142,6 @@ fn url_large_text_button(
         if egui::Button::image_and_text(texture_id, ReUi::small_icon_size(), text)
             .ui(ui)
             .on_hover_cursor(egui::CursorIcon::PointingHand)
-            .on_hover_text(url)
```

Comment:
why? was it annoying?

related: maybe we should delay showing the tooltips by a bit of time? Right now they show up as soon as the mouse stops, but we can add a small delay, see https://github.com/emilk/egui/pull/3245

What says @martenbjork ?

Reply:
He's had strong words _against_ showing URL in tooltips: "Bad UX 😠" (emoji emphasis mine). Thus this change.

Reply:
Yeah, I think showing URLs as tooltips is lazy.

- URLs take brain CPU to parse
- They assume that the user is familiar with the domain and path and that it is meaningful for them

If we want tooltips here, they should be something user friendly like "View documentation" or "Learn more on the Rerun website".

But I'd argue that labeled buttons rarely should have tooltips. In most cases, the label + icon should be enough to understand the action.